openpype/settings/entities/schemas/projects_schema/schema_project_maya.json

```
@@ -153,6 +153,23 @@
         },
         {
             "type": "dict",
+            "collapsible": true,
+            "key": "update_instances",
```

This is not explicit enough in maya context. It should be clearer what it relates too

```suggestion
            "key": "update_publishable_frame_range",
```